sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/CaseService.java

```diff
@@ -72,6 +73,36 @@ public Case buildGeneratedCase() {
         .build();
   }
 
+  public Case buildGeneratedCaseDE() {
+    firstName = faker.name().firstName();
+    lastName = faker.name().lastName();
+
+    return Case.builder()
+        .firstName(firstName)
+        .lastName(lastName)
+        .caseOrigin("IM LAND")
+        .dateOfReport(LocalDate.now().minusDays(1))
+        .externalId(UUID.randomUUID().toString())
+        .disease("COVID-19")
+        .responsibleRegion(RegionsValues.VoreingestellteBundeslander.getName())
+        .responsibleDistrict(DistrictsValues.VoreingestellterLandkreis.getName())
+        .responsibleCommunity(CommunityValues.VoreingestellteGemeinde.getName())
+        .placeOfStay("ZUHAUSE")
```

Comment:
can we vary this data?

Reply:
In the Cases we never used place of stay different than "Home". It is because "Facility" needs more fields to fill so it can't be random. We also haven't enum for that - never needed